vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go

```diff
@@ -96,7 +96,7 @@ func TestDialURL(t *testing.T) {
 			ts.TLS = &tls.Config{Certificates: []tls.Certificate{cert}}
 			ts.StartTLS()
 
-			tlsConfigCopy := utilnet.CloneTLSConfig(tc.TLSConfig)
+			tlsConfigCopy := tc.TLSConfig
```

Comment:
this is actually wrong (it was a bug in the upstream PR, as noted in https://github.com/kubernetes/kubernetes/pull/41758#discussion_r123909365)

once we're on go1.8 alone, the suggested solution there could work. until then, we need to continue cloning with utilnet.CloneTLSConfig